### DISCUSSION TITLE
English Patch?

### DISCUSSION THREAD

**Discussion by wegotaproblem** [planetarian/damecon-browser#4] [General]
As the title says, I'm curious about the English patch for kccacheproxy or some other equivalent. Is there already some kind of integration for this built into Damecron? If not is it possible to do so? Any help would be appreciated, thanks.

**Comment by planetarian** [comment]
It should function normally through KCCP, but I haven't tested this as I don't use it myself. KCCP stuff is on my list of things to verify though, so maybe soon I'll have more answers.

**Comment by planetarian** [comment]
As of v0.6.0, KCCP is now integrated and I have confirmed that the EN patch functions properly.
